app/src/test/java/org/simple/clinic/home/overdue/OverdueUpdateTest.kt

```diff
@@ -54,4 +55,43 @@ class OverdueUpdateTest {
             hasEffects(ShowOverdueAppointments(overdueAppointments, isDiabetesManagementEnabled = true))
         ))
   }
+
+  @Test
+  fun `when current facility is loaded and overdue list changes feature is disabled, then load overdue appointments without patients with no phone numbers`() {
```

Comment:
There seems to be a typo in the test name, we are loading patients **with phone number** when the overdue list changes feature is disabled right?

Reply:
Yeah, which is why it says - 'without patients with no phone numbers'

Reply:
Alright, I think I lost my mind. Clearly, I did not see that on Friday 😂